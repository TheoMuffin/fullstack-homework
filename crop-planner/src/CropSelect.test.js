import { shallow } from 'enzyme'

import CropSelect from './CropSelect'

describe('<CropSelect />', () => {
  it('renders', () => {
    expect(shallow(
      <CropSelect
        selectedCrop={{value: 1, label: 'My Crop'}}
        allCrops={[
          {value: 1, label: 'My Crop'},
          {value: 2, label: 'My Second Crop'},
        ]}
        onChange={jest.fn()}
      />
    )).toMatchSnapshot()
  })
})
