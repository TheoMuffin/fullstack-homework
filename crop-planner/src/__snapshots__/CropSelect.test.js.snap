// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CropSelect /> renders 1`] = `
<StateManager
  className="react-select react-select--table"
  classNamePrefix="react-select"
  defaultInputValue=""
  defaultMenuIsOpen={false}
  defaultValue={null}
  onChange={[MockFunction]}
  options={
    Array [
      Object {
        "label": "My Crop",
        "value": 1,
      },
      Object {
        "label": "My Second Crop",
        "value": 2,
      },
    ]
  }
  value={
    Object {
      "label": "My Crop",
      "value": 1,
    }
  }
/>
`;
